README.md

```diff
@@ -1 +1,17 @@
 # blog-app
+> In this blog-app, I will create a fully functional website that will show the list of posts and allows readers to interact with them by adding comments and liking posts.
+
+Projects list
+I will be building the Blog app for three weeks. Here is the list of projects that will guide me through the steps described above. 
+- Project 1: Setup and controllers.
+- Project 2: Controllers specs.
+- Project 3: Creating a data model.
+- Project 4: Processing data in models.
+- Project 5: Views.
+- Project 6: Forms
+- Project 7: Validations, Model specs, and n+1 problems.
+- Project 8: Add Devise.
+- Project 9: Add authorization rules.
+- Project 10: Integration specs for Views.
+- Project 11: Add API endpoints.
+- Exercise: API documentation.
```

Comment:
- [ ] Kindly use the microverse README template by customizing it with your own information. This will make your README file look more professional.
![image](https://user-images.githubusercontent.com/65068771/172727988-14dc579d-bec7-4564-b9ce-9e57d8cac1c3.png)

Reply:
Thanks for your review,it have been updated